### PR TITLE
Add description about dump-format

### PR DESCRIPTION
**Pull request by rimms**
I tried to write about dump-format.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -35,6 +35,42 @@ Usage
...
+  - ``storage``
+
+    - ``weight`` : Weights of each features and each labels in data inputted so far.
+- recommender (inverted_index)
+
+  - ``index``
+
+    - ``storage``
+
+      - ``inv``: IDs of each features
```
Comment (kmaehashi) [inline README.rst]: Is this correct? As far as I tested I didn't see anything in this block.

File: README.rst [hunk 2]
```diff
@@ -35,6 +35,42 @@ Usage
...
+
+Format
+======
+
+- common
+
+  - ``weights``
+
+    - ``version_number`` : Version of model. This value will be updated by MIX.
+    - ``document_frequencies`` : Frequency of each features in data inputted so far.
```
Comment (kmaehashi) [inline README.rst]: `each features` -> `each feature` (should be in singular form)

File: README.rst (hunk 1 above)
Comment (rimms) [inline README.rst]: Your right. I fixed.

**Comment by rimms** [comment]
@kmaehashi Thanks for review, fixed.

**Comment by kmaehashi** [comment]
:+1:

**Pull request merged**